mkdocs/tests/plugin_tests.py

```diff
@@ -20,13 +20,21 @@ class DummyPlugin(plugins.BasePlugin):
     )
 
     def on_pre_page(self, content, **kwargs):
```

Comment:
This is unrelated to the PR but would it be OK to change this event to on_page_content to more closely match its docstring?